src/CommunityToolkit.Aspire.Hosting.OpenTelemetryCollector/OpenTelemetryCollectorSettings.cs

```diff
@@ -0,0 +1,27 @@
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Settings for the OpenTelemetry Collector
+/// </summary>
+public class OpenTelemetryCollectorSettings
+{
+    /// <summary>
+    /// The version of the collector, defaults to latest
+    /// </summary>
+    public string CollectorVersion { get; set; } = "latest";
+
+    /// <summary>
+    /// The image of the collector, defaults to ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib
+    /// </summary>
+    public string CollectorImage { get; set; } = "ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib";
```

Comment:
I feel like this should be split to registry/image like we do with other container resources. And is there a reason you're using ghcr.io not Docker Hub?

Reply:
ghcr is the recommended default since the limits went into dockerhub.

Do you have an example of the pattern other integrations use for the registry split.

Reply:
https://github.com/CommunityToolkit/Aspire/blob/0c0afa121b5c4078310f1f13eca49cbe3c77c609/src/CommunityToolkit.Aspire.Hosting.Ollama/OllamaContainerImageTags.cs#L5-L6